Pod/Classes/Core/public/WistiaAPI.swift

```diff
@@ -123,6 +125,43 @@ public class WistiaAPI {
 }
 
 extension WistiaAPI {
+    //MARK: - thumbnail
+    
+    /**
+     Get thumbnail of a given movie hash id (does not need an API token).
+     
+     See [Wistia Working with images - https://wistia.com/doc/working-with-images#using_oembed)
+     
+     - parameter forHash: the hashId that identifies the movie
+     - parameter imageWidthPoints: desired width of the image in points (the function converts to pixels taken into account the screen's scale)
+     - parameter completionHander: The block to invoke when the API call completes. The block takes 2 arguments: the requested thumbnail and the error status code of the request (if an error occured)
+     */
+    public static func thumbnail(forHash movieHashedId: String, imageWidthPoints: CGFloat = 0, completionHander: @escaping (_ thumbnail: WistiaMedia.Thumbnail?, _ error: WistiaAPIError?) -> () ){
+        //convert from points to pixels or take default value of 640 pixels
+        let widthPixels = imageWidthPoints > 0 ? Int(imageWidthPoints * UIScreen.main.scale) : 640
+        //construct the url
+        let url = "http://fast.wistia.net/oembed?url=http://home.wistia.com/medias/$$movieHasedId$$?embedType=async&videoWidth=$$imageWidth$$".replacingOccurrences(of: "$$movieHasedId$$", with: movieHashedId).replacingOccurrences(of:"$$imageWidth$$", with: String(widthPixels))
```

Comment:
Absolutely not. Its a leftover from some debugging I did. We should proceed with Sting interpolation.